dependencies.props

```diff
@@ -1,14 +1,43 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <!-- Package versions used as toolsets --> 
-    <PropertyGroup> 
-        <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage> 
-        <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion> 
-    </PropertyGroup> 
-    <ItemGroup>
-        <XmlUpdateStep Include="BuildTools">
-            <Path>$(MSBuildThisFileFullPath)</Path>
-            <ElementName>FeedTasksPackageVersion</ElementName>
-            <PackageId>$(FeedTasksPackage)</PackageId>
-        </XmlUpdateStep>
-    </ItemGroup>
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+  <!-- For detailed explanation of the reference versions see:
+
+       https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+       https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+       These ref versions are pulled from https://github.com/dotnet/versions.
+  -->
+  <PropertyGroup>
+    <BuildToolsCurrentRef>c6906dcb2309085e414c5b9691de84fbe2bf5c86</BuildToolsCurrentRef>
+  </PropertyGroup>
+
+  <!-- Package versions used as toolsets --> 
+  <PropertyGroup> 
+    <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage> 
+    <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion> 
+  </PropertyGroup> 
+
+  <!-- Package dependency verification/auto-upgrade configuration. -->
+  <PropertyGroup>
+    <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
+    <DependencyBranch>master</DependencyBranch>
+    <CurrentRefXmlPath>$(MSBuildThisFileFullPath)</CurrentRefXmlPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <RemoteDependencyBuildInfo Include="BuildToolsFeed">
```

Comment:
If `standard` starts taking BuildTools updates (https://github.com/dotnet/corefx/blob/2bd69b97b763e0acdfa082eed1a281a5f5afffa5/dependencies.props#L152-L156) then this will just have to change back to `BuildTools` to be accurate. I think the name should be kept the same.

This itemspec also needs to match the `BuildTools` part of `BuildToolsCurrentRef`--that's how the updater knows where the commit hash is.

Reply:
The version of the task feed, and the version of buildtools, would be different.  Does that not matter?

Reply:
The `RemoteDependencyBuildInfo` basically tells VersionTools to fetch all package versions available at the given dotnet/versions commit hash for the given repo, then updaters pick up specific package versions from that list. There's no problem using the same `RemoteDependencyBuildInfo` for multiple packages with different versions.

Reply:
Makes sense, thanks